amcl/src/amcl_node.cpp

```diff
@@ -1273,17 +1282,22 @@ AmclNode::laserReceived(const sensor_msgs::LaserScanConstPtr& laser_scan)
   if(resampled || force_publication)
   {
     // Read out the current hypotheses
+    move_base_msgs::amcl_data dp; //data_msg
+
     double max_weight = 0.0;
     int max_weight_hyp = -1;
     std::vector<amcl_hyp_t> hyps;
     hyps.resize(pf_->sets[pf_->current_set].cluster_count);
+    int total_num_samples = 0;
     for(int hyp_count = 0;
         hyp_count < pf_->sets[pf_->current_set].cluster_count; hyp_count++)
     {
+      move_base_msgs::cluster tC; //cluster to push back to data_msg.clusters
```

Comment:
Why tC for variable name?  camelCase doesn't match the surrounding style.  It's better to keep your code style similar to the code you are modifying